### DISCUSSION TITLE
사용자의 스크롤 위치 기억하기 (History.scrollRestoration에 대해서) - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#36] [Announcements]
# 사용자의 스크롤 위치 기억하기 (History.scrollRestoration에 대해서) - 정현수 기술 블로그

사용자의 스크롤 위치를 기억하기 위해서 사용해야 하는 browser API

https://junghyeonsu.com/posts/history-scroll-restoration/

**Comment by dev-93** [comment]
safari에서도 잘 작동하나요? https://okky.kr/articles/1384416 글처럼 ios 사파리일경우에 왜 안되는지 알고싶습니다 ㅠㅠ
